tools/topic_tools/package.xml

```diff
@@ -28,6 +28,8 @@
   <build_depend>xmlrpcpp</build_depend>
 
   <run_depend>message_runtime</run_depend>
+  <run_depend>python-numpy</run_depend>
+  <run_depend>python-texttable</run_depend>
```

Comment:
I have removed the dependency on these packages.

I don't mind slow releasing of ros_comm, and putting tools that can be used in general cases on upstream package is better, I think. Tools in lowstream package will soon obsolete when the maintainer's status changes.
(We have some packages as shared package in my lab, but some of them are not used after the author student graduates.)